bigchaindb/block.py

```diff
@@ -4,11 +4,16 @@
 
 import rethinkdb as r
 
+import bigchaindb
 from bigchaindb import Bigchain
+from bigchaindb.monitor import Monitor
+
 
 
 logger = logging.getLogger(__name__)
 
+c = Monitor()
```

Comment:
Give a more descriptive name to `c`, I think `monitor` is fine. (Yup, we will give a more descriptive name also to `b` one day)